modules/client-handler/src/main/java/org/apache/ignite/client/handler/JdbcQueryEventHandlerImpl.java

```diff
@@ -210,9 +213,14 @@ private static SqlProperties createProperties(
                 throw new AssertionError("Unexpected jdbc statement type: " + stmtType);
         }
 
+        // TODO: https://issues.apache.org/jira/browse/IGNITE-24021
+        // Converts schema name to its canonical form.
+        String schemaNameInCanonicalForm = QualifiedName.fromSimple(defaultSchemaName).objectName();
```

Comment:
```suggestion
        // TODO: https://issues.apache.org/jira/browse/IGNITE-24021
        //  Replace this with using correct implementation of `IgniteNameUtils.parseSimpleName`, when it will be fixed.
        String schemaNameInCanonicalForm = QualifiedName.fromSimple(defaultSchemaName).objectName();
```
```suggestion
        // TODO: https://issues.apache.org/jira/browse/IGNITE-24021
        // Converts schema name to its canonical form.
        String schemaNameInCanonicalForm = QualifiedName.fromSimple(defaultSchemaName).objectName();
```